generation_config.yaml

```diff
@@ -6,7 +6,7 @@ libraries:
     name_pretty: Cloud Spanner
     product_documentation: https://cloud.google.com/spanner/docs/
     client_documentation: https://cloud.google.com/java/docs/reference/google-cloud-spanner/latest/history
-    api_description: is a fully managed, mission-critical,  relational database service that offers transactional consistency at global scale, \nschemas, SQL (ANSI 2011 with extensions), and automatic, synchronous replication \nfor high availability.\n\nBe sure to activate the Cloud Spanner API on the Developer's Console to\nuse Cloud Spanner from your project.
+    api_description: is a fully managed, mission-critical,  relational database service that offers transactional consistency at global scale, schemas, SQL (ANSI 2011 with extensions), and automatic, synchronous replication for high availability. Be sure to activate the Cloud Spanner API on the Developer's Console to use Cloud Spanner from your project.
```

Comment:
Isn't this file auto-generated by the code generator? Which means that these changes will be overwritten during the next code generation?

Reply:
No. autogeneration only updates the following fields. Not the actual file. 

* googleapis_commitish
* gapic_generator_version

https://github.com/googleapis/java-spanner/blob/main/.github/scripts/update_generation_config.sh#L95-L103

Reply:
ack, thanks for clarifying